### DISCUSSION TITLE
Can EZRA Bible be build inside our Flutter App?

### DISCUSSION THREAD

**Discussion by legorreta** [ezra-bible-app/ezra-bible-app#458] [Q&A] [answered]
Hi guys,

I want to start by saying how cool is this project.

We are currently building the App to control our night light for Christian families. 

Will a Flutter Developer be able to use the EZRA project to power the Bible tab in our App?

Thank you,
Christian Legorreta
++

**Comment by tobias-klein** [comment, marked as answer]
Hi Christian!

I don't know much about Flutter and how it works. Therefore it's hard for me to make a general statement whether integrating Ezra will work for you ...

Ezra consists of a frontend and a backend. The frontend is fully JavaScript/HTML/CSS and it should be rather straightforward to integrate that in whatever web view context you can imagine. The backend also contains native components (C++) and therefore the backend maybe more difficult to handle. Also, you need to consider the link between frontend and backend. This is currently handled via asynchronous IPC calls and may need to be ported to your technology depending on how it works.

Have a look [here](https://apidocs.ezrabibleapp.net/) to find out more about the design/structure of Ezra Bible App.

**Reply by legorreta** [reply]
Thank you Tobias, this information I am sure will be useful for the developer. Blessings. 😊✨
